global.json

```diff
@@ -0,0 +1,8 @@
+{
+  "sdk": {
+    "version": "2.1.101"
```

Comment:
Now that we're past preview2, can this be upgraded to `2.1.300-preview2-008533`, the 2.1-preview2 SDK? This will match source-build. Otherwise we'll need to add a patch.

Reply:
As discussed in https://github.com/dotnet/sdk/pull/2113#discussion_r183776898, updating this isn't necessary--a temporary patch is reasonable and we'll have a global.json updater in source-build pretty soon.